docs/source/usersguide/basics.rst

```diff
@@ -0,0 +1,213 @@
+:ref:`scripts_validate` script.
+
+.. _XML: http://www.w3.org/XML/
+
+Creating Input Files
+--------------------
+
+.. currentmodule:: openmc
+
+The simplest option to create input files is to simply write them from scratch
+using the :ref:`XML format specifications <io_file_formats_input>`. This
+approach will feel familiar to users of other Monte Carlo codes such as MCNP and
+Serpent, with the added bonus that the XML formats feel much more
+"readable". Alternatively, input files can be generated using OpenMC's
+:ref:`Python API <pythonapi>`, which is introduced in the following section.
+
+----------
+Python API
+----------
+
+OpenMC's :ref:`Python API <pythonapi>` defines a set of functions and classes
+that roughly correspond to elements in the XML files. For example, the
+:class:`openmc.Cell` Python class directly corresponds to the
+:ref:`cell_element` in XML. Each XML file itself also has a corresponding class:
+:class:`openmc.Geometry` for ``geometry.xml``, :class:`openmc.Materials` for
+``materials.xml``, :class:`openmc.Settings` for ``settings.xml``, and so on. To
+create a model then, one creates instances of these classes and then uses the
+``export_to_xml()`` method, e.g. :meth:`Geometry.export_to_xml`. Most scripts
+that generate a full model will look something like the following:
+
+.. code-block:: Python
+
+   # Create materials
+   materials = openmc.Materials()
+   ...
+   materials.export_to_xml()
+
+   # Create geometry
+   geom = openmc.Geometry()
+   ...
+   geom.export_to_xml()
+
+   # Assign simulation settings
+   settings = openmc.Settings()
+   ...
+   settings.export_to_xml()
+
+One a model has been created and exported to XML, a simulation can be run either
+by calling :ref:`scripts_openmc` directly from a shell or by using the
+:func:`openmc.run()` function from Python.
+
+If you have never used Python before, the prospect of learning a new code *and*
+a programming language might sound daunting. However, you should keep mind in
+mind that there are many substantial benefits to using the Python API,
+including:
+
+- The ability to define dimensions using variables.
+- Availability of standard-library modules for working with files.
+- An entire ecosystem of third-party packages for scientific computing.
+- Ability to create materials based on natural elements or uranium enrichment
+- :ref:`Automated multi-group cross section generation <pythonapi_mgxs>`
+- Convenience functions (e.g., a function returning a hexagonal region)
+- Ability to plot individual universes as geometry is being created
+- A :math:`k_\text{eff}` search function (:func:`openmc.search_for_keff`)
+- Random sphere packing for generating TRISO particle locations
+  (:func:`openmc.model.pack_trisos`)
+- A fully-featured :ref:`nuclear data interface <pythonapi_data>`.
+
+.. tip:: Users are strongly encouraged to use the Python API to generate input
+         files and analyze results.
+
+Identifying Objects
```

Comment:
Perhaps it is just me, but this kind of seems out of place here, though I understand that it is very important for "legacy" users of other codes.